Typo in ManticoreEVM#last_return

dwhjames opened the pull request:
Encountered the following trace:

```
Traceback (most recent call last):
  …, in <module>
    m.last_return()
  File "…/manticore/ethereum.py", line 906, in last_return
    return state.platform.last_return
AttributeError: 'EVMWorld' object has no attribute 'last_return'
```

It appears this bug was introduced in commit ed29a22f

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/trailofbits/manticore/886)
<!-- Reviewable:end -->

dwhjames commented:
`ManticoreEVM#last_return` attempts to call `EVMWorld#last_return`
https://github.com/trailofbits/manticore/blob/9307475af5f52d641e97b8208ac53ea4812e5940/manticore/ethereum.py#L903-L906
but the property is actually
https://github.com/trailofbits/manticore/blob/9307475af5f52d641e97b8208ac53ea4812e5940/manticore/platforms/evm.py#L2049-L2051

yan commented:
@dwhjames thanks for this pr! It looks like a similar issue exists in [evm.py](https://github.com/trailofbits/manticore/blob/master/manticore/platforms/evm.py#L2243). Update that too and we'll merge?

dwhjames commented:
So there is, @yan!

I’ve updated that and squashed both changes into a new commit https://github.com/trailofbits/manticore/pull/886/commits/8752b48f4fbf638ce5d9aa235d491c1fe3b03e1a
